dragonfly_schema/window_parameter.py

```diff
@@ -90,3 +91,98 @@ class RepeatingWindowRatio(BaseModel):
         description='An optional number to create a single vertical separation '
             'between top and bottom windows.'
     )
+
+
+class DetailedRectangularWindows(BaseModel):
```

Comment:
I think `RectangularWindows` will be just as good of a name.

Reply:
I think this is a good call. I'll implement it over on dragonfly-core and then implement it here.